.github/workflows/docker-build.yml

```diff
@@ -274,6 +274,15 @@ jobs:
           path: resoto.com
           token: ${{ secrets.SOME_CI_PAT }}
 
+      - name: Update resource data models
+        continue-on-error: true
+        shell: bash
+        run: |
+          PSK= RESOTOCORE_ANALYTICS_OPT_OUT=true docker-compose up -d
+          cd resoto.com/docs/reference/data-models
+          find . -mindepth 2 -type d -name img -prune -exec rm -rf {} \;
+          python ${{ github.workspace }}/resoto.com/tools/export_model_images.py
```

Comment:
This will create a couple of hundred file changes for every commit, or?
Can we do better?